Jenkinsfile

```diff
@@ -27,6 +27,7 @@ final def isStdBranch = (env.BRANCH_NAME == "master" || env.BRANCH_NAME.startsWi
 final def tsbiImage = "artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/jdk11-svc-builder:2.9.27-20220331162145"
 final def jdk17Image= "artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/jdk17-svc-builder:2.9.27-20220331162145"
 final def podLabel = "component-runtime-${UUID.randomUUID().toString()}".take(53)
+final boolean isOnMasterOrMaintenanceBranch = env.BRANCH_NAME == 'master' || env.BRANCH_NAME.startsWith('maintenance/')
```

Comment:
use the `isStdBranch` variable, no need to create a new one.

Reply:
Good catch, missed it :)